proto/control-api/build.rs

```diff
@@ -9,21 +9,44 @@ use std::{error::Error, fs::File, io::ErrorKind};
 /// ```
 /// in the root of the project.
 #[cfg(feature = "grpc")]
-fn main() -> Result<(), Box<dyn Error>> {
-    const GRPC_DIR: &str = "src/grpc/";
-    const GRPC_SPEC_FILE: &str = "src/grpc/api.proto";
-    const OUT_FILES: [&str; 2] = ["src/grpc/api.rs", "src/grpc/api_grpc.rs"];
+fn compile_grpc_specs() -> Result<(), Box<dyn Error>> {
+    const GRPC_DIR: &str = "src/grpc";
 
-    println!("cargo:rerun-if-changed={}", GRPC_SPEC_FILE);
-    for filename in &OUT_FILES {
-        println!("cargo:rerun-if-changed={}", filename);
-    }
+    let proto_names: Vec<String> = std::fs::read_dir(GRPC_DIR)
+        .unwrap()
```

Comment:
Why unwrapping when you have permissive `-> Result<(), Box<dyn Error>>`? Just use the `?` operator.